### DISCUSSION TITLE
Increase available storage by increasing disk space of worker nodes

### DISCUSSION THREAD

**Discussion by amrap030** [longhorn/longhorn#6372] [Q&A]
Hello together,

I have a K3S Cluster running on Proxmox with 3 Master and 2 Worker Nodes. In Proxmox I would like to give my 2 Worker Nodes more storage, but it doesn't reflect in longhorn. I would expect that my longhorn UI now shows more available disk space. Is that even possible? If yes, what do I need to do?

Thanks in advance!

**Comment by mantissahz** [comment]
Hi @amrap030,
I'm not sure about the way you would try to give more storage but Longhorn will have a disk monitor to update the disk storage when the node status has been updated.
